### DISCUSSION TITLE
Using xterm.js

### DISCUSSION THREAD

**Discussion by 0xdevz** [bajrangCoder/acode-plugin-acodex#52] [Q&A]
how i can use xterm.js for terminal instead termux

**Comment by bajrangCoder** [comment]
> how i can use xterm.js for terminal instead termux

Xterm.js is not a backend, it's only a frontend, you will need any server backend to use it. Check out : https://xtermjs.org

**Reply by 0xdevz** [reply]
yeah i know because with termux you cant install every package its need EACCES permission 
![Screenshot_20240204_005719_Termux](https://github.com/bajrangCoder/acode-plugin-acodex/assets/86024087/92a0447d-48fd-4bc6-a49f-9780607da09f)

**Reply by bajrangCoder** [reply]
Then run the server on any cloud or in termux there are two solutions to fix it:

1. Install these npm packages in the termux home directory
2. Add this `--no-bin-links` flag while installing package in the internal storage

**Reply by 0xdevz** [reply]
the ``--no-bin-links`` worked thank you
